docs/source/_static/css/custom.css

```diff
@@ -103,3 +106,7 @@ h3.admonition-title::before {
     mask-image: var(--icon-admonition-default);
     mask-repeat: no-repeat;
 }
+/* Apply class selector from inline page.html */
```

Comment:
This comment doesn't really make sense outside the context of this PR. I would remove it and/or provide a more accurate comment that describes what the class actually does when applied to an html element.
Example: `/* Hides an element by removing it from the document flow */`